wpengine/faustjs.org
# Style Post template

These changes add styles to the post template, including any tags and/or categories after the content.

## Testing

1. Go the latest atlas preview environment for this pull request.
2. Go to `/blog` in order to view a list of our existing posts.
3. Observe that the posts are rendering properly.
